lib/matplotlib/contour.py

```diff
@@ -249,7 +249,7 @@ def get_label_width(self, lev, fmt, fsize):
                 self._mathtext_parser = mathtext.MathTextParser('bitmap')
             img, _ = self._mathtext_parser.parse(lev, dpi=72,
                                                  prop=self.labelFontProps)
-            lw = img.get_width()  # at dpi=72, the units are PostScript points
+            lw, _ = np.shape(img)  # at dpi=72, the units are PostScript points
```

Comment:
Couldn't really follow the image definition, but are you sure the width is the number of rows (not columns)?

Reply:
oops, fixed